source/dbo/Stored Procedures/Unmatched Specimens/uspGenerateUnmatchedSpecimensLinkedNotifications.sql

```diff
@@ -38,23 +38,20 @@ BEGIN TRY
 	SELECT al.ReferenceLaboratoryNumber
 		,al.EventType AS NotificationLinkReason
 		,rs.RejectionDate
-		,rs.UserDisplayName as [User]
+		,rs.UserDisplayName AS [User]
 		,al.NotificationID
-		,n.NotificationStatus
-		,p.NhsNumber
-		,p.Dob AS BirthDate
-		,p.FamilyName
-		,p.GivenName
-		,sex.[Label] AS [Sex]
-		,p.[Address]
-		,p.PostcodeToLookup AS Postcode
-		,tbs.PHECCode AS RegionCode
+		,CASE WHEN rr.Denotified = 0 THEN 'No' ELSE 'YES' END AS Denotified
```

Comment:
Noticed a bug here - I was getting whether the record was closed rather than denotified as before